test/localeModels/gbt/maxTaskPar.prediff

```diff
@@ -40,7 +40,7 @@ case $target in
   linux*|cray-xc|cray-xe|cygwin*)
     case $locModel-$tasks in
       numa-qthreads)
-        ucp=$( $CHPL_HOME/util/printchplenv --make |
+        ucp=$( $CHPL_HOME/util/printchplenv --make --path |
```

Comment:
I think we mean for this one to use $CHPL_HOME/util/printchplenv --all --internal--make 
For one thing --path overrides --make... and for another the grep is checking for an internal variable so we definitely need --internal.